includes/class-wc-taxjar-integration.php

```diff
@@ -269,26 +271,70 @@ public function get_settings( $current_section = '' ) {
 					'type'  => 'title',
 					'desc'  => '',
 				),
-				array(
-					'title'   => __( 'API Token', 'wc-taxjar' ),
-					'type'    => 'text',
-					'desc'    => __( '<p><a href="' . $this->app_uri . 'account#api-access" target="_blank">Click here</a> to get your API token.</p>', 'wc-taxjar' ),
-					'default' => '',
-					'id'      => 'woocommerce_taxjar-integration_settings[api_token]'
-				),
-				array(
-					'type' => 'sectionend',
-				),
+
 			);
 
+			if ( $api_token_valid ) {
+				$connected_email = $this->post_or_setting( 'connected_email' );
+
+				if ( $connected_email ) {
+					array_push( $settings, array(
+						'title' => '',
+						'type'  => 'title',
+						'desc'  => '<div class="taxjar-connected><span class="dashicons dashicons-yes-alt"></span><p>' . $connected_email . '</p></div>',
+						'id'	=> 'connected-to-taxjar'
+					) );
+				} else {
+					array_push( $settings, array(
+						'title' => '',
+						'type'  => 'title',
+						'desc'  => '<div class="taxjar-connected><span class="dashicons dashicons-yes-alt"></span><p>Connected To TaxJar</p></div>',
+						'id'	=> 'connected-to-taxjar'
+					) );
+				}
+
+				array_push( $settings, array(
+					'title' => '',
+					'type'  => 'title',
+					'desc'  => __( '<button id="disconnect-from-taxjar" name="disconnect-from-taxjar" class="button-primary" type="submit" value="Disconnect">Disconnect From TaxJar</button><p><a href="#" id="connect-manual-edit">Click here to edit API Token manually.</a></p>', 'wc-taxjar' ),
```

Comment:
I would rephrase this link text to avoid "click here" type wording.

Maybe just `Edit API Token` would be sufficient?